debezium-core/src/main/java/io/debezium/transforms/debezium-timestamp-converter/src/main/java/org/telmate/SMT/OptionalTypeFieldSchema.java

```diff
@@ -0,0 +1,13 @@
+package org.telmate.SMT;
+
+import org.apache.kafka.connect.data.Schema;
+
+/**
+ * Created by svegesna on 9/29/17.
+ */
+public interface OptionalTypeFieldSchema {
```

Comment:
Could you add a description of that interfaces purpose and remove the creation date?